### PR TITLE
Feature/intergrate dropdown graphs

### PR DESCRIPTION
**Pull request by alanblee**
# Description
Fixes #PR_NUMBER 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
## Change Status
- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback
## Has This Been Tested
- [ ] Yes
- [x] No
- [ ] Not necessary
## Does this require communication between DS and WEB?
- [ ] Yes, and I have already reached out to someone
- [x] Yes, but I still need to reach out to someone
- [ ] No
## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least two peers
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Comment by alanblee** [comment]
Note about the LineGraph, once we have the proper endpoints, shouldn't need those axios request inside the useEffect.

File: src/components/common/Graphs/LineGraph.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        .get(`https://b-ds.citrics.dev/viz/${state2}`)
+        .then(({ data }) => setData2(JSON.parse(data).data));
+    }
+    if (state3) {
+      axios
+        .get(`https://b-ds.citrics.dev/viz/${state3}`)
+        .then(({ data }) => setData3(JSON.parse(data).data));
+    }
+  }, [state, state2, state3]);
+
```
Comment (ddhorstman) [inline src/components/common/Graphs/LineGraph.js:26]: I wonder if this might make more sense as three separate useEffect-s.

Comment (alanblee) [reply src/components/common/Graphs/LineGraph.js:26]: This is just for testing purposes. Ideally we'll have all the data when the cities are selected so we won't need to do any axios requests.

**Review by ddhorstman — APPROVED** [review]
Looks good! I think there may be a fancier way to do your useEffect in LineGraph.js but from a functional standpoint everything seems solid. Thanks for doing this!

File: src/styles/LESS/comparison-page.less [hunk 2]
```diff
@@ -1,23 +1,25 @@
 .comparison-container {
-  .background-image("/compare_screen_full.jpg");
+  // .background-image("/compare_screen_full.jpg");
   display: flex;
-  justify-content: center;
+  // justify-content: center;
+  align-items: center;
```
Comment (berachele) [inline src/styles/LESS/comparison-page.less:5]: I'll have to look at the home page to see what changes you made -- pretty sure it was to accommodate for the comparison graph so no big deal. Thanks for implementing the comparison graph and tabs for the demographics!

**Review by berachele — APPROVED** [review]

File: src/components/common/Graphs/LineGraph.js (hunk 1 above)
Comment (ddhorstman) [reply src/components/common/Graphs/LineGraph.js:26]: Ahh gotcha! Definitely not worth optimizing now then.

**Review by lyndsiWilliams — APPROVED** [review]
I like the tabs much better than the dropdown idea!

**Pull request merged**